src/modules/user/user.controller.ts

```diff
@@ -428,6 +430,97 @@ export class UserController {
     }
   }
 
+  @UseGuards(ThrottlerGuard)
+  @Post('/unblock-account')
+  @HttpCode(200)
+  @ApiOperation({
+    summary: 'Request account unblock',
+  })
+  @Public()
+  async requestAccountUnblock(@Body() body: RequestAccountUnblock) {
+    try {
+      return await this.userUseCases.sendAccountUnblockEmail(body.email);
```

Comment:
`return` directly or assign to variable and `return` the constant

Reply:
If we return without the await (I know it looks odd) the catch in the controller can not get the error so I will asssign it to a variable.